packages/tc-schema-sdk/data-accessors/relationship-types.js

```diff
@@ -0,0 +1,40 @@
+const type = require('./type');
+
+module.exports = {
+	// todo move to object rest parameters for options when upgrading node
+	accessor(options = {}) {
+		const types = this.rawData.getTypes();
```

Comment:
Question - does this return only relationshipTypes that have been user define in the schema, or does it also return ones which are generated from analysing relationship properties on types. My instinct is to return autogenerated ones too.

Reply:
No in here, but actually returns relationships which resolved from type properties on following logic -- here: https://github.com/Financial-Times/treecreeper/pull/69/files/05f4aff2c1cece2790f97b9d69aa4a7fe31a48df#diff-dc4679c40157e5b6ce606145a4df7c11R13-R34

Reply:
ok I think I understand.